nav2_bringup/launch/multi_tb3_simulation_launch.py

```diff
@@ -60,17 +61,18 @@ def generate_launch_description():
     # Declare the launch arguments
     declare_world_cmd = DeclareLaunchArgument(
         'world',
-        default_value=os.path.join(bringup_dir, 'worlds', 'world_only.model'),
+        default_value=os.path.join(aws_dir, 'worlds', 'small_warehouse', 'small_warehouse.world'),
         description='Full path to world file to load')
 
     declare_simulator_cmd = DeclareLaunchArgument(
         'simulator',
         default_value='gazebo',
-        description='The simulator to use (gazebo or gzserver)')
+        description='The simulator to use (gazebo or gzserver)',
+    )
 
     declare_map_yaml_cmd = DeclareLaunchArgument(
         'map',
-        default_value=os.path.join(bringup_dir, 'maps', 'turtlebot3_world.yaml'),
+        default_value=os.path.join(aws_dir, 'maps', '005', 'map.yaml'),
```

Comment:
Please move this map into `maps/` in bringup, that way we can make modifications over time if needed (then update to use here and in the 1 robot sim launch file)

Reply:
Also, delete the turtlebot3 world maps from here